doozerlib/distgit.py

```diff
@@ -1396,6 +1391,19 @@ def update_distgit_dir(self, version, release, prev_release=None):
                 if in_mod_block:
                     continue
 
+                # If we are injecting environment variables, remove any line
+                # from the Dockerfile that may be trying to set the same name. This is also
+                # necessary for us to clean up our older env injections if this is a dist-git only
+                # Dockerfile.
+                if line.lower().startswith('env'):
+                    skip_line = False
+                    for var_name in env_vars:
+                        if f' {var_name}=' in line:
+                            skip_line = True
+                            break
+                    if skip_line:
+                        continue
```

Comment:
Define a variable to break from outer loop, yuk... How about
```suggestion
                    if any(f' {var_name}=' in line for var_name in env_vars):
                        continue
```